macOS/DuckDuckGo/Tab/UserScripts/Subscription/SubscriptionAppStoreRestorer.swift

```diff
@@ -80,13 +80,13 @@ struct DefaultSubscriptionAppStoreRestorer: SubscriptionAppStoreRestorer {
                 PixelKit.fire(PrivacyProPixel.privacyProRestorePurchaseStoreFailureOther, frequency: .legacyDailyAndCount)
             }
             switch error {
-            case .missingAccountOrTransactions:
+            case .missingAccountOrTransactions, .pastTransactionAuthenticationError:
```

Comment:
@miasma13 this is for you to review:
I tested the app behaviour for this scenario: https://app.asana.com/0/0/1209427500692943 and it was ok, except for the alert we show, currently we show "something went wrong try again later" but the `pastTransactionAuthenticationError ` is generally unrecoverable so I've changed the alert to a more final "subscription not found", what do you think?